pages/index.js

```diff
@@ -2,36 +2,55 @@ import HeadObject from "../components/head";
 import Nav from "../components/nav";
 import Scene from "../components/scene";
 import Balancer from "react-wrap-balancer";
+import Link from "next/link";
 
 export default function Home() {
   return (
     <div className="dark:text-white dark:bg-black">
-      <HeadObject />
-      <a href="https://hackclub.com"><img src="https://assets.hackclub.com/flag-orpheus-top.svg" className="absolute top-0 left-8 w-1/4 md:w-[14%]" /></a>
+      {/* <HeadObject />
+      <Link href="https://hackclub.com">
+        <img
+          src="https://assets.hackclub.com/flag-orpheus-top.svg"
+          className="absolute top-0 left-8 w-1/4 md:w-[14%]"
+        />
+      </Link> */}
+
+      <div>
+        <a class="banner mx-auto ml-2 z-50" href="https://hackclub.com/">
+          <img
+            src="https://contribute.hackclub.com/images/flag-orpheus-top.png"
+            alt="Hack Club Logo"
+          ></img>
+        </a>
+      </div>
+
       <main>
-        <section
-          id="hero"
-          className="flex flex-col items-center relative"
-        >
-          {/* <Nav /> */}
+        <section id="hero" className="flex flex-col items-center relative">
           <div className="my-24 flex flex-col gap-4 items-center text-center z-10">
-            <h1 className="text-9xl">
+            <h1 className="text-6xl sm:text-9xl">
```

Comment:
Can you explain why this change is needed?

Reply:
to make it work for mobile

Reply:
It seemed to work on mobile before, do you have screenshots of it not working?

Reply:
See above.